web/src/app/schedules/ScheduleRuleDeleteDialog.tsx

```diff
@@ -64,16 +61,21 @@ export default function ScheduleRuleDeleteDialog(
       title={`Remove ${startCase(props.target.type)} From Schedule?`}
       subTitle={`This will remove all rules, as well as end any active or future on-call shifts on this schedule for ${props.target.type}: ${data.schedule.target.target.name}.`}
       caption='Overrides will not be affected.'
+      loading={deleteRuleStatus.fetching}
+      errors={deleteRuleStatus.error ? [deleteRuleStatus.error] : []}
       confirm
       onSubmit={() => {
-        deleteRule({
-          variables: {
+        deleteRule(
+          {
             input: {
               target: props.target,
               scheduleID: props.scheduleID,
               rules: [],
             },
           },
+          { additionalTypenames: ['ScheduleTargetInput', 'Schedule'] },
```

Comment:
Only output types will need to be refreshed

```suggestion
          { additionalTypenames: ['Schedule'] },
```